src/presentation/controllers/configuration-controller.ts

```diff
@@ -43,4 +43,14 @@ export class ConfigurationController extends BaseController {
     }
   }
 
+  @PostRequest('/reset')
```

Comment:
A minor note here. `/reset` in the context of configurations could sound like returning what has been configured to defaults rather than clearing the cache.